### PR TITLE
zh, en: refine docs for deploying TiDB on GKE

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

refine docs for deploying TiDB on GKE

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [x] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/deploy-tidb-from-kubernetes-gke.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-tidb-from-kubernetes-gke/']
 
 > **警告：**
 >
-> 对于生产环境，不要使用此方式进行部署。
+> 对于生产环境，请参考我们的[在 GCP 上通过 Kubernetes 部署 TiDB 集群](deploy-on-gcp-gke.md)。
```
Comment (qiancai) [inline zh/deploy-tidb-from-kubernetes-gke.md:25]:
```suggestion
> 本文中部署说明仅用于测试目的，**不要**直接用于生产环境。如果要在生产环境部署，请参阅[在 GCP 上通过 Kubernetes 部署 TiDB 集群](deploy-on-gcp-gke.md)。
```

File: en/deploy-tidb-from-kubernetes-gke.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ It takes you through the following steps:
 
 > **Warning:**
 >
-> This is for testing only. DO NOT USE in production!
+> For production use, please consult [Deploy TiDB on GCP GKE](deploy-on-gcp-gke.md).
```
Comment (qiancai) [inline en/deploy-tidb-from-kubernetes-gke.md:25]:
```suggestion
> This tutorial is for testing purposes only. **Do not** follow it in production environments. For production environments, see the instructions in [Deploy TiDB on GCP GKE](deploy-on-gcp-gke.md).
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Comment by qiancai** [comment]
After confirming with @DanielZhangQD, the TiDB Operator v1.0 is not maintained anymore so we can remove the needs-cherry-pick-1.0 label to skip the cherry-pick to v1.0.

**Pull request merged**